spec/controllers/api/v3/notices_controller_spec.rb

```diff
@@ -16,9 +16,9 @@
     post :create, legit_body, legit_params
     notice = Notice.last
     expect(JSON.parse(response.body)).to eq({
-      'id' => notice.id.to_s,
-      'url' => app_problem_url(app, notice.problem)
-    })
+                                              'id' => notice.id.to_s,
+                                              'url' => app_problem_url(app, notice.problem)
+                                            })
```

Comment:
There's a bunch more instances of this specific thing in the ./spec folder. If these multi-line hash literals can be just 2-space indented, then I think this PR would be ready to go.